src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -634,6 +634,8 @@ public virtual NamedTypeSymbol TupleUnderlyingType
         /// </remarks>
         internal abstract bool IsManagedType { get; }
 
+        bool ITypeSymbol.IsUnmanagedType => !IsManagedType;
```

Comment:
>bool ITypeSymbol.IsUnmanagedType => !IsManagedType; [](start = 8, length = 51)

Would it make sense to replace ```IsManagedType``` API with ```IsUnmanagedType``` API within the compiler? It is confusing when internal APIs do not align with public APIs